omod/src/main/java/org/openmrs/module/coreapps/web/resource/LatestObsResource.java

```diff
@@ -31,12 +27,22 @@
 import org.openmrs.module.webservices.rest.web.resource.api.PageableResult;
 import org.openmrs.module.webservices.rest.web.resource.impl.EmptySearchResult;
 import org.openmrs.module.webservices.rest.web.resource.impl.NeedsPaging;
+import org.openmrs.module.webservices.rest.web.response.ResponseException;
 import org.openmrs.module.webservices.rest.web.v1_0.resource.openmrs1_11.ObsResource1_11;
 import org.openmrs.module.webservices.rest.web.v1_0.resource.openmrs1_8.PatientResource1_8;
 
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Vector;
+
 @Resource(name = RestConstants.VERSION_1 + "/latestobs", order = 1, supportedClass = Obs.class, supportedOpenmrsVersions = {
 		"1.8.*", "1.9.*", "1.10.*", "1.11.*", "1.12.*", "2.0.*", "2.1.*", "2.2.*" })
```

Comment:
Can we also add platform 2.3?

Reply:
Done

Reply:
I think we've started just doing 2.* in many cases, not sure if this is "best practice" or not.